content/docs/searching.md

```diff
@@ -139,30 +140,30 @@ The **Filters** list is a space-separated list of wildcard expressions that cmd.
 * You can also exclude certain file patterns by prefixing the filter with a `!`;
 for example, **Filters:  `!*.bin *.*`** will exclude files matching `*.bin` from the search results, but include any other filename.
 * As of Notepad++ v8.2, you can also exclude particular folders from the search: The exclusion operator is always `!` at the beginning of the expression, so in order to distinguish folder from file, `\` should be used as prefix of the folder name/pattern, following `!`. That allows the exclusion of the directories under the root directory you want to search (the 1st level of matched directories).
-If users need to exclude folders with the same name (or names matched the specific pattern) in all levels, the `+` should be put between `!` and `\` to exclude them recursively. For example:
-    * `!\tests` will not search any files in the `tests` folder
-    * `!\bin*` will not search any files in the `bin` folder or `bin64` folder (or any other directory that matches `bin*`)
-    * `!+\log*` will _recursively_ not search any files in folders that start with log (so directories like `.\log`, `.\logs`, `.\other\logfiles`, `.\many\layers\deep\log` will all be excluded from the search)
+If users need to exclude folders with the same name (or names matching the specific pattern) in all levels, the `+` should be put between `!` and `\` to exclude them recursively. For example:
+    * `!\tests` will not search any files in the `tests` folder,
+    * `!\bin*` will not search any files in the `bin` folder or `bin64` folder (or any other directory that matches `bin*`),
+    * `!+\log*` will _recursively_ not search any files in folders that start with log (so directories like `.\log`, `.\logs`, `.\other\logfiles`, `.\many\layers\deep\log` will all be excluded from the search).
 
-    *Note*: "inclusion of folder" is not allowed, and such pattern will be ignored
+    *Note*: Inclusion of subfolders of excluded folders is not allowed, and any such pattern will be ignored.
```

Comment:
No.  There is no way to use the filter to include subfolders.
The syntax `!\tests` is valid to exclude a subfolder, but `\tests` will _not_ cause a new folder to be included, which was the point of that statement.  By saying "of excluded folders", it implies that you are trying to undo a previous exclusion-filter.

Reply:
Isn't this how you would use an inclusion statement, though? You start with everything from a given directory and exclude folders. The only folders you could include then are subfolders of excluded folders.

I was confused as I read this because this is about *filters* whose only point is to exclude things, so I thought inclusion would be about exactly that: partially reversing exclusions, or in other words: excluding everything but a part of a directory.
If, to go with your example, `!\tests` isn't part of the filter, `\tests` is never needed as it is included in the search like every other folder in the directory. However, `\tests\subfolder` would make some sense to reinclude an exiled part.

Inclusion of folders isn't allowed, correct, that's what the original line stated, and I got that. However, as folders by themselves are never in need of being included, I think it would make sense to write explicitly of subfolders.

Reply:
> this is about filters whose only point is to exclude things
 
I guess that's not quite the way I think of filters. Because the filter `*.cpp` is about what to include, rather than what to exclude -- though I guess the implication is "exclude everything, then just include `*.cpp`"

> The only folders you could include then are subfolders of excluded folders.

Unless you're non-recursive.  (Though again, I guess you'd might think of it as "non-recursive is essentially excluding all subdirectories")

Maybe a different wording:
```
*Note*: There is no opposite ("inclusive") version of the folder-exclusion syntax.  This means that if your search is not using `☐ In all sub-folders`, you cannot add a specific subfolder to include; it also means that if you exclude a subfolder, you cannot tell it to include a specific deeper-subfolder (so `!\skip \skip\exceptThis` will _not_ work to skip the `\skip` subfolder but to still search the `\skip\exceptThis` deeper-subfolder).
```